docs/install_and_run.rst

```diff
@@ -6,25 +6,40 @@ Install OS2borgerPC server image
 
 Get the most recent OS2borgerPC server image from the archive, e.g.
 
-http://bibos-admin.magenta-aps.dk/archive/os2borgerpc-server-0.1.0.iso
+http://bibos-admin.magenta-aps.dk/archive/TO_BE_SPECIFIED.iso
```

Comment:
I feel like we should either decide on this URL or remove it altogether. (I think the documentation for the normal OS2borgerPC installation just says that Magenta will provide an appropriate installation image.)

Reply:
Given our new policy of not providing images for public download, it should be removed - good point.

Reply:
Rewritten.